### PR TITLE
Update dependencies + more stuff

### PR DESCRIPTION
**Pull request by mariozugaj**
This PR is part of the learning (hack) hour.

It adds/changes a few things:
- updates `rubocop` and other dependencies,
- adds a `.rubocop.yml` file (this was to test out the config, but I think it could stay),
- updates `README.md` (some info didn't apply to this gem),
- adds a `Gemfile.lock` file,
- cleans up `default.yml` to suppress the warnings,
- disables `RSpec/NestedGroups` cop (we are usually disabling it or not following it).

This will also possibly mean updating dependencies in all projects where's it's used -> that's the next step.

### CODE REVIEW

**Review by jaimevelaz — APPROVED** [review]

**Comment by iconpin** [comment]
👏  Great! This also fixes some problems we were having with the RiskScreener (it uses Ruby 2.7.1 and we were getting weird `NoMethodError` errors for standard library classes 😵 ).

I know Maksim wants to review the changes, so I'll wait until then to beg you to merge 😂 .

**Review by uncoder — APPROVED** [review]
Looks great! 👏 🚀

**Pull request merged**
